test/functional/commands/general/mobile-command-e2e-specs.js

```diff
@@ -6,19 +6,48 @@ import { initDriver } from '../../helpers/session';
 chai.should();
 chai.use(chaiAsPromised);
 
-describe('mobile: shell', function () {
+describe('mobile', function () {
   let driver;
   before(async function () {
-    driver = await initDriver(APIDEMOS_CAPS);
+    delete APIDEMOS_CAPS.app;
+    driver = await initDriver({
+      ...APIDEMOS_CAPS,
+
+      // For deeplinking to work, it has to run a session in a native
+      // context but it doesn't matter what native app is run so just
+      // run io.appium.settings for simplicity
+      appPackage: 'io.appium.settings',
+      appActivity: '.Settings',
+    });
   });
   after(async function () {
     await driver.quit();
   });
-  it('should call execute command without proxy error, but require relaxed security flag', async function () {
-    try {
-      await driver.execute('mobile: shell', {command: 'echo', args: ['hello']});
-    } catch (e) {
-      e.message.should.match(/Original error: Appium server must have relaxed security flag set in order to run any shell commands/);
-    }
+  describe('mobile:shell', function () {
+    it('should call execute command without proxy error, but require relaxed security flag', async function () {
+      try {
+        await driver.execute('mobile: shell', {command: 'echo', args: ['hello']});
+      } catch (e) {
+        e.message.should.match(/Original error: Appium server must have relaxed security flag set in order to run any shell commands/);
+      }
+    });
+  });
+  describe('mobile:deepLink', function () {
+    it('should be able to launch apps using Instant Apps', async function () {
+      try {
+        await driver.execute("mobile: deepLink", {url: 'https://www.realtor.com/realestateandhomes-search/San-Jose_CA', package: 'com.move.realtor'});
+      } catch (e) {
+        e.message.should.match(/unable to resolve intent/i);
```

Comment:
Elsewhere we ensure that there is a particular emulator available which is capable of handling the case under test. See, for example, https://github.com/appium/appium-android-driver/blob/master/test/functional/chrome-e2e-specs.js

Reply:
I don't think there will be any emulators that can do this, actually. There's a setting that you have to enable on a device.

Reply:
That should be noted, so the test _can_ be run to make sure it works.